src/constants.py

```diff
@@ -13,3 +13,18 @@
 SDAI_TOKEN_ADDRESS = Web3.to_checksum_address(
     "0x83F20F44975D03b1b09e64809B757c47f942BEeA"
 )
+
+# Time limit after which Coingecko Token List is re-fetched (in seconds)
+TOKEN_LIST_RELOAD_TIME = 86400
+
+# Time in seconds of 45 hours
+COINGECKO_TIME_LIMIT = 162000
+
+# Buffer time interval to allow 5-minutely Coingecko prices to be fetched
+BUFFER_TIME = 600
```

Comment:
if this constant is specific to coingecko, maybe add that to the name.

what is the difference of this buffer compared to QUERY_BUFFER_TIME below?